packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.spec.js

```diff
@@ -0,0 +1,42 @@
+import { shallowMount } from "@vue/test-utils";
+import SfStoreLocator from "@/components/organisms/SfStoreLocator/SfStoreLocator.vue";
```

Comment:
same with alaises

Reply:
This is a test, do you really want full path here?

Reply:
youre right